apps/src/code-studio/pd/regional_partner_mini_contact/RegionalPartnerMiniContact.jsx

```diff
@@ -26,16 +30,32 @@ const styles = {
   },
   intro: {
     paddingBottom: 10
+  },
+  select: {
+    maxWidth: '500px'
```

Comment:
```suggestion
    maxWidth: 500
```
Note that React _usually_ [interprets number values in inline styles as pixel values](https://react-cn.github.io/react/tips/style-props-value-px.html) (unlike CSS where you really ought to always specify units) and it's idiomatic to lean on this.